packages/babel-traverse/src/path/index.ts

```diff
@@ -302,21 +289,43 @@ const methods = {
 Object.assign(NodePath_Final.prototype, methods);
 
 if (!process.env.BABEL_8_BREAKING && !USE_ESM) {
+  // String(x) is workaround for rollup
+
   // @ts-expect-error babel 7 only
   NodePath_Final.prototype.arrowFunctionToShadowed =
-    // workaround for rollup
     // @ts-expect-error babel 7 only
     NodePath_conversion[String("arrowFunctionToShadowed")];
+
+  Object.assign(NodePath_Final.prototype, {
+    // @ts-expect-error Babel 7 only
+    has: NodePath_introspection[String("has")],
```

Comment:
What is `String(...)` for?

Reply:
This is for rollup, otherwise it will show that the export cannot be found.

Reply:
Can you add a comment? :)